content/getting-started/setup.md

```diff
@@ -18,7 +18,15 @@ As a user, nearly all of your interactions with Cloud Foundry will be through th
     cf -v
     ```
 
-1. Log in by running
+1. **If you log in using your agency's account system**, log in by running
+
+    ```bash
+    cf login -a api.cloud.gov --sso
+    ```
+    
+    Then follow the passcode link to get your one time code and enter it to log in.
```

Comment:
"Registration" is misleading because no account is being registered... How about "One-Time Authentication Code"?